src/index.ts

```diff
@@ -28,27 +27,24 @@ export default async function resolveLocal (
   const spec = parsePref(wantedDependency.pref, opts.prefix)
   if (!spec) return null
 
-  const dependencyPath = normalize(path.relative(opts.prefix, spec.fetchSpec))
```

Comment:
What will be the effect of this?

Reply:
I just moved this to another file. This normalizes the path to be included in the package id